src/rez/bind/python.py

```diff
@@ -65,13 +70,23 @@ def bind(path, version_range=None, opts=None, parser=None):
             if pypath not in builtin_paths.values():
                 builtin_paths[dirname] = pypath
 
+    # add DLLs
+    basedir = os.path.dirname(exepath)
+    if platform_.name == "windows":
```

Comment:
Safely ignore this one, as this might be beyond the scope of this PR/change. 

`platform` module should be directly available and `from rez.utils.platform_ import platform_` purely for this purpose might not be needed.

```
>>> import platform
>>> platform.system()
'Linux'
```